assets/constants.ts

```diff
@@ -5310,6 +5310,21 @@ export = {
   "OSK_FUNCTIONS_TAB": "functionsTab",
   "OSK_LETTERS_TAB": "lettersTab",
 
+  // Name to human readable form mapping of the position of terms object.
+  "POSITION_OF_TERMS_MAPPING": [{
+    "name": "lhs",
+    "humanReadableName": "on LHS"
```

Comment:
Should you actually expand LHS in the human readable name?

Reply:
Done.